### PR TITLE
fixed small grammatical and typing mistakes

### PR DESCRIPTION
**Pull request by nitinsultania**
# Issue #138
corrected some grammatical and typing mistakes 

### Changes
- 3 files changed
  - 3 commits

### CODE REVIEW

File: docs/spec-concepts.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ A legal professional analyzes a contract to determine the frequently used/standa
...
 
 ## Step 3: Generation of the Template Parser
 The [Cicero Open Source project](https://github.com/accordproject/cicero) contains code that can automatically generate a parser from the annotated template text (template grammar) and the associated template data model. The parser generation is completely automatic and supports customization of types and nested grammars.
 
 ## Step 4: Create a Smart Clause
-The generated template parser can now be used to dynamically edit and validate source smart clause text (potentially using code completion, error reporting etc). The editor technology can be embedded on a webpage, or executed as a SaaS service, or run within an IDE.
+The generated template parser can now be used to dynamically edit and validate source smart clause text (potentially using code completion, error reporting etc.). The editor technology can be embedded on a webpage, or executed as a SaaS service, or run within an IDE.
```
Comment (irmerk) [inline docs/spec-concepts.md]: This can be removed.

File: docs/spec-concepts.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ A legal professional analyzes a contract to determine the frequently used/standa
 
 ## Step 2: Data Modeling
 
-Concept Template Data Model. The variables and expressions in a template are expressed in terms of a typed data model, that captures all the concepts of relevance to the clause. The data model technology allows importing concepts from namespaces, allowing concepts to be shared across templates.
+Concept Template Data Model. The variables and expressions in a template are expressed in terms of a typed data model that captures all the concepts of relevance to the clause. The data model technology allows importing concepts from namespaces, allowing concepts to be shared across templates.
```
Comment (irmerk) [inline docs/spec-concepts.md]: This can be removed.

**Comment by nitinsultania** [comment]
can you please help me how to add a DCO

**Comment by irmerk** [comment]
I believe you'll be running `git commit --amend --signoff` and then force pushing after that is corrected.

**Comment by nitinsultania** [comment]
added signed off

**Pull request closed without merging**